src/utils/output.ts

```diff
@@ -3,14 +3,25 @@ import { Output as Console } from '@jfonx/console-utils';
 export class Output {
   constructor(private silent = false) {}
 
+  /**
+   * Crash the process with a non-zero exit code
+   */
+  private static exitProcessWithError() {
+    process.exit(1);
+  }
+
   info(message: string) {
     if (this.silent) return;
     Console.info(message);
   }
 
-  error(message: string, exitProces = false) {
-    if (this.silent) return;
-    Console.error(message, exitProces);
+  error(message: string, exitProcess = false) {
+    if (!this.silent) {
+      Console.error(message);
```

Comment:
The `Console.error` method does take an optional 2nd arg for whether or not to exit, which is what was previously being used. However, in that case the method exits using `process.exit()` without passing an argument, which defaults to `0`. The result is then that an error will cause a "SUCCESS" exit instead of a non-zero ERROR exit, which could cause pipelines to succeed when they shouldn't.